README.md

```diff
@@ -383,7 +385,8 @@ RestClient.post('www.example.net', 'abc')    # ===> "abc\n"
 
 ### Dynamically evaluated raw responses recorded with `curl -is`
 
-    `curl -is www.example.com > /tmp/www.example.com.txt`
+`curl -is www.example.com > /tmp/www.example.com.txt`
```

Comment:
This causes the `curl` request to show up in a code block with backticks around it.